hydra-node/src/Hydra/Chain/Direct/Tx.hs

```diff
@@ -310,6 +310,8 @@ closeTx vk closing (slotNo, utcTime) openThreadOutput =
       & addOutputs [headOutputAfter]
       & addExtraRequiredSigners [verificationKeyHash vk]
       & setValidityUpperBound slotNo
+      -- TODO: check if this is ok? we are using the same slot for lower/upper tx bound
+      & setValidityLowerBound slotNo
```

Comment:
This is unrealistic that this works. As said in another comment we need some slack between lower and upper bound. We can use the current time for the lower bound (add an additional argument to the function) and the same upper bound as before.